### DISCUSSION TITLE
latest ubuntu and raspberry 400 using latest GUI install 2 TB SSD  SYNCED but NOTHING happening!

### DISCUSSION THREAD

**Discussion by dralibak** [Chia-Network/chia-blockchain#6140] [Support]
latest ubuntu and raspberry 400 using latest GUI install, 2 TB SSD, SYNCED but NOTHING happening!
Plots are chosen on SSD, in parallel.  ALL show 1 percent x 3 days post complete sync now.  no progress, no plot , pool key, or farm key !!

nothing is happening! 

Wallet empty. everything looks connected ... but no harvesting, farming, plot activity, wallet activity being shown.
is this because of PORT? i didn't get any errors on that. again i am using graphical GUI install.

What do I do ?

**Comment by tiny0little** [comment]
are you plotting on raspberry pi?

**Comment by dralibak** [comment]
Yes raspberry pi 400 with latest Ubuntu desktop arm64 with Gui Installer


Thank you.


Sincere,

Ali Bakhshalizadeh

**Comment by dralibak** [comment]
Also using 2 TB External SSD


Thank you.


Sincere,

Ali Bakhshalizadeh

**Comment by tiny0little** [comment]
I think raspberry pi won't be suitable for plotting, due to weak CPU and low on RAM. you need something much-much more powerful to be successful. take a look at reference plotting hardware here - https://github.com/Chia-Network/chia-blockchain/wiki/Reference-Plotting-Hardware

in one case they use laptop cpu in NUC tiny desktop, I would not recommend this option

you can farm on raspberry pi, as farming does not take as much cpu/memory

**Comment by BrandtH22** [comment]
@dralibak , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
